pkgs/applications/audio/plexamp/darwin.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, undmg
+, pname
+, version
+, meta
+, ...
+}:
+
+stdenv.mkDerivation {
+  inherit pname version meta;
+  src =
+    if stdenv.isAarch64 then
+      (fetchurl
+        {
+          url = "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-${version}-arm64.dmg";
+          sha256 = "sha256-VmADu5vC2SZs930mgSw0uF69xSsPfTy/1feBYTHM55g=";
+        })
+    else
+      (
+        fetchurl
+          {
+            url = "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-${version}.dmg";
+            sha256 = "sha256-gXJySo/8LwDkyR0mJNou5BD+7oxx6Crmcatf/uyMTjQ=";
+          }
+      );
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  dontFixup = true;
```

```suggestion
  dontBuild = true;
```

disabling patches here just complicates things when users want to apply patches locally and it doesn't break things. Probably the same with configure.

Why not fixup?

---

sorry, t I'm not very familiar with building a nixpkg